docs/kubernetes_migration_guide.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Kubernetes Migration Guide
+layout: docs
+---
+
+_Document outline_
+
+## Migrating to Kubernetes
+- Are you migrating to Kubernetes?
+- This guide is for you.
+- Tilt helps developments teams do development, that are already deploying to Kubernetes now, or are planning to do so in the future.
+- Many teams focus on the production aspect of Kubernetes. They often don't realize that it has downstream impacts on development. This guides puts both together in perpsective.
+
+## Production migration
+- Here are some resources on production migration of Kubernetes.
```

Comment:
I'd even say "There are lots of people who will do this, so we're not going to." instead of trying to maintain something that's not our competency.